internal/github/pullRequests.go

```diff
@@ -3,38 +3,58 @@ package github
 import (
 	"context"
 	"fmt"
+	"log"
+	"strings"
 
 	"github.com/google/go-github/v60/github"
+
+	"github.com/chia-network/github-bot/internal/config"
 )
 
-// FindCommunityPRs obtains non-teammember PRs
-func FindCommunityPRs(owner string, repo string, teamMembers map[string]bool, githubClient *github.Client) ([]*github.PullRequest, error) {
+// FindCommunityPRs obtains PRs based on provided filters
+func FindCommunityPRs(repos []config.CheckRepo, teamMembers map[string]bool, githubClient *github.Client) ([]*github.PullRequest, error) {
 	var finalPRs []*github.PullRequest
 	opts := &github.PullRequestListOptions{
 		State:     "open",
 		Sort:      "created",
 		Direction: "desc",
 		ListOptions: github.ListOptions{
-			Page:    0,
+			Page:    1,
 			PerPage: 100,
 		},
 	}
-	for {
-		opts.ListOptions.Page++
-		pullRequests, resp, err := githubClient.PullRequests.List(context.TODO(), owner, repo, opts)
-		if err != nil {
-			return finalPRs, fmt.Errorf("error listing pull requests: %w", err)
+
+	for _, fullRepo := range repos {
+		log.Println("Checking repository:", fullRepo.Name)
+		parts := strings.Split(fullRepo.Name, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repository name - must contain owner and repository: %s", fullRepo.Name)
 		}
+		owner, repo := parts[0], parts[1]
 
-		for _, pullRequest := range pullRequests {
-			user := *pullRequest.User.Login
-			if !teamMembers[user] {
-				finalPRs = append(finalPRs, pullRequest)
+		for {
+			pullRequests, resp, err := githubClient.PullRequests.List(context.TODO(), owner, repo, opts)
+			if err != nil {
+				return nil, fmt.Errorf("error listing pull requests for %s/%s: %w", owner, repo, err)
 			}
-		}
 
-		if resp.NextPage == 0 {
-			break
+			for _, pullRequest := range pullRequests {
+				if *pullRequest.Number < fullRepo.MinimumNumber {
+					continue
+				}
```

Comment:
If we're going to be checking for the minimum PR number in the API call loop, you can actually `break` here to svae on API requests and move on to the next repo, because the API request options say to sort the PRs by created, descending, so the PR numbers should always be descending.

Reply:
Changed the continue to a break